exercises/concept/resistor-color/.docs/instructions.md

```diff
@@ -31,7 +31,10 @@ The goal of this exercise is to create a way:
 - to convert the numerical value into a string representing color
 - to list the different band colors
 
-For tasks number two and three, you will need external crates [`enum-iterator`](https://docs.rs/enum-iterator/1.1.1/enum_iterator/) and [`int-enum`](https://docs.rs/int-enum/0.4.0/int_enum/), which are included in this exercise's `Cargo.toml`. Be sure to check the crates' documentation to learn how to use them.
+For task number one, you will need the external crate [`int-enum`](https://docs.rs/int-enum/0.4.0/int_enum/) and for task number three, you'll need [`enum-iterator`](https://docs.rs/enum-iterator/1.1.1/enum_iterator/).
+They're both already included in this exercise's `Cargo.toml`. 
+Be sure to check the crates' documentation to learn how to use them.
+You can look into the default implementation of the Debug trait for Enums to complete task number two.
```

Comment:
I hope this doesn't lead students to try to look for the source code behind the implementation, just to look at what it does. I don't have any particular improvement to this sentence in mind right now. It can be reevaluated later if it is a problem.